specification/cognitiveservices/data-plane/WebSearch/v1.0/WebSearch.json

```diff
@@ -0,0 +1,1211 @@
+        },
+        "provider": {
+          "description": "The source of the creative work.",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Thing"
+          }
+        },
+        "text": {
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "Intangible": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/Thing"
+        }
+      ],
+      "type": "object"
+    },
+    "MediaObject": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/CreativeWork"
+        }
+      ],
+      "type": "object",
+      "properties": {
+        "contentUrl": {
+          "description": "Original URL to retrieve the source (file) for the media object (e.g the source URL for the image).",
+          "readOnly": true,
+          "type": "string"
+        },
+        "hostPageUrl": {
+          "description": "URL of the page that hosts the media object.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "width": {
+          "description": "The width of the source media object, in pixels.",
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32"
+        },
+        "height": {
+          "description": "The height of the source media object, in pixels.",
+          "readOnly": true,
+          "type": "integer",
+          "format": "int32"
+        }
+      }
+    },
+    "WebMetaTag": {
+      "description": "Defines a webpage's metadata.",
+      "x-ms-discriminator-value": "Web/MetaTag",
+      "type": "object",
+      "properties": {
+        "name": {
+          "description": "The metadata.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "content": {
+          "description": "The name of the metadata.",
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "Article": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/CreativeWork"
+        }
+      ],
+      "type": "object"
+    },
+    "RankingRankingItem": {
+      "description": "Defines a search result item to display",
+      "x-ms-discriminator-value": "Ranking/RankingItem",
+      "type": "object",
+      "required": [
+        "answerType"
+      ],
+      "properties": {
+        "answerType": {
+          "description": "The answer that contains the item to display. Use the type to find the answer in the SearchResponse object. The type is the name of a SearchResponse field.",
+          "type": "string",
+          "default": "WebPages",
+          "enum": [
+            "WebPages",
+            "Images",
+            "SpellSuggestions",
+            "News",
+            "RelatedSearches",
+            "Videos",
+            "WebPageGroupings",
+            "Computation",
+            "TimeZone"
+          ],
+          "x-ms-enum": {
+            "name": "AnswerType",
+            "modelAsString": false
```

Comment:
It feels like that list of values in the enum could be expanded later. Perhaps we should set `modelAsString` to `true` -- this will reduce breaking changes later if a new value is added .